index.html

```diff
@@ -0,0 +1,180 @@
+            >
+              <div class="">
+                <img class="icon" src="icons/forum.svg" alt="" />
+              </div>
+              <p class="program text-center mt-sm-3 ms-3">Forum to Discuss</p>
+              <p class="program-description text-white">
+                Adobe Creative Cloud (Photoshop, Illustrator, XD), Sketch,
+                Figma, Affinity Designer, GIMP
+              </p>
+            </div>
+
+            <div
+              class="block my-1 d-flex flex-md-column align-items-center px-3 py-3 me-md-2"
+            >
+              <div class="">
+                <img class="icon" src="icons/ignite.svg" alt="" />
+              </div>
+              <p class="program text-center mt-sm-3 ms-3">HTML/CSS Editors</p>
+              <p class="program-description text-white">
+                Tools for editing HTML/CSS code, making web design easier.
+                Examples: Sublime Text, Atom, Brackets, Dreamweaver, Notepad++.
+              </p>
+            </div>
+          </div>
+
+          <button class="join p-4 mt-5">Join WebDev Bootcamp 2023</button>
+          <p class="desktop-buttone">
+            <a href="" class="desktop-button">SEE THE WHOLE PROGRAM</a>
+          </p>
+        </div>
+      </article>
+      <artical class="speakers mt-5 px-4 py-5 bg-white">
+        <h2 class="main-program text-center text-black">Featured Speakers</h2>
+        <h2 class="baseline text-center text-white fw-bold"></h2>
+        <img src="" alt="" />
+        <div class="main-speakers my-5"></div>
+
+        <p
+          id="seemore"
+          class="morex w-100 py-2 d-flex d-lg-none justify-content-center align-items-center"
+        >
+          More
+          <img class="arrow ms-2" src="icons/arrowdown.svg" alt="" />
+        </p>
+      </artical>
+      <article
+        class="card partners mt-5 bg-dark d-sm-none d-md-block text-white"
+      >
+        <h2 class="partner text-center mt-5">Partner</h2>
+        <h2 class="baseline fw-bold"></h2>
+        <div class="companies">
+          <div class="row d-flex justify-content-center mb-3">
+            <img class="col-4 py-4" src="icons/mozilla.svg" alt="" />
+            <img class="col-4" src="icons/google.svg" alt="" />
+            <img class="col-4" src="icons/naver.svg" alt="" />
+          </div>
+          <div class="row d-flex justify-content-center pb-4">
+            <img class="col-6" src="icons/daumkakao.svg" alt="" />
+            <img class="col-6" src="icons/airbnb.svg" alt="" />
+          </div>
+        </div>
+      </article>
+      <article class="d-sm-none d-md-block">
+        <div class="reserved d-flex justify-content-center mt-4">
+          <img class="logo-end" src="img/summitlogo.jpeg" alt="" />
+          <p class="footertxt ende text-center">
+            <strong>2020 Global Construction Engineers<br /></strong>
+            Some Rights reserved.
```

Comment:
- as suggested by the previous reviewer your should sure the mobile version of the home page ends at the show more button and has more similar features to the original design
- and properly align your featured speaker's section to fit the mobile version because it's overlapping.

your design:
![image](https://user-images.githubusercontent.com/60612329/226094860-0e35343e-c1e7-49c2-8342-9d8224f683cd.png)
![image](https://user-images.githubusercontent.com/60612329/226094874-35ba908c-be69-4a05-b43b-3c682099e0ab.png)

cindy's design:
![image](https://user-images.githubusercontent.com/60612329/226094889-9fd304bf-bf1e-4c6e-a626-713cd33e4ae8.png)